src/signals/incident/components/IncidentPreview/components/DateTime/DateTime.tsx

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2018 - 2021 Gemeente Amsterdam
+import format from 'date-fns/format'
+import { capitalize } from 'shared/services/date-utils'
+import nl from 'date-fns/locale/nl'
+
+import type { Incident } from 'types/incident'
+import type { FC } from 'react'
+
+interface DateTimeProps {
+  value: Incident['timestamp']
+  incident?: Incident
+}
+
+const getValue = (value: Incident['timestamp'], incident?: Incident) => {
+  if (value === null) {
+    return 'Nu'
```

Comment:
is it possible to use constants for 'Nu' and 'Eerder'?

Reply:
Ah, good one. I'll make a type for that.